README.md

```
@@ -563,6 +571,7 @@ The script waits for all namespaces to be fully deleted before completing.
 > run this script. Instead, manually delete only the OSAC-specific resources:
 > ```bash
 > $ oc delete -k overlays/<project-name>
+> $ oc delete -k base/osac-operator/config/console-proxy-kube-system
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)